titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/workflow/step/TaskEvictionStep.java

```diff
@@ -87,7 +87,7 @@ public TaskEvictionStep(EvictionServiceClient evictionServiceClient,
                 .collect(Collectors.toMap(
                         TaskRelocationPlan::getTaskId,
                         p -> {
-                            String message = String.format("%s: reasonCode=%s, plannedRelocationTime=%s",
+                            String message = String.format("%s\nreasonCode=%s plannedRelocationTime=%s",
```

Comment:
Is this the new line character on purpose? I think we should keep all messages in a single line.

Reply:
It was on purpose, I thought it looked better. I'll do one line though.